### DISCUSSION TITLE
Car data

### DISCUSSION THREAD

**Discussion by markste-in** [theOehrly/Fast-F1#117] [Q&A]
As far as I know ff1 has no car data in it?

Is there a good source where you could get technical data on the cars?
For example
- dimensions
- weight
- what engine
- etc...

**Comment by theOehrly** [comment]
No, FastF1 does not provide that information. And I don't know of any source that can provide you with that data, if you can get the data at all. 
For weight, I'd just use the regulated minimum weight. You'll never get accurate values for the current generation of cars. And for older cars, that data will be difficult to find.
